static/app/views/settings/organizationMembers/organizationMembersWrapper.tsx

```diff
@@ -161,17 +164,27 @@ function renderInviteMembersButton({
   );
 
   return disabled ? (
-    <Hovercard
-      body={
-        <FeatureDisabled
-          featureName={t('Invite Members')}
-          features="organizations:invite-members"
-          hideHelpToggle
-        />
-      }
-    >
-      {action}
-    </Hovercard>
+    isSsoRequired ? (
+      <Tooltip
+        title={t(
+          `Your organization must use its single sign-on provider to register new members.`
+        )}
+      >
```

Comment:
I think you can add `skipWrappper` here
```suggestion
      <Tooltip
        skipWrapper
        title={t(
          `Your organization must use its single sign-on provider to register new members.`
        )}
      >
```